k8s/templates/clear-old-tasks-cronjob.yaml

```diff
@@ -0,0 +1,78 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "studio.fullname" . }}-clear-old-tasks-job-config
+  labels:
+    tier: job
+    app: {{ template "studio.fullname" . }}
+    chart: {{ .Chart.Name }}
+    release: {{ .Release.Name }}
+data:
+  DJANGO_LOG_FILE: /var/log/django.log
+  DATA_DB_HOST: {{ template "cloudsql-proxy.fullname" . }}
+  DATA_DB_PORT: "5432"
+  MPLBACKEND: PS
+  RUN_MODE: k8s
+  RELEASE_COMMIT_SHA: {{ .Values.studioApp.releaseCommit | default "" }}
+  BRANCH_ENVIRONMENT: {{ .Release.Name }}
+  AWS_BUCKET_NAME: {{ .Values.studioApp.gcs.bucketName }}
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ template "studio.fullname" . }}-clear-old-tasks-job-secret
+  labels:
+    app: {{ template "studio.fullname" . }}
+    chart: {{ .Chart.Name }}
+    release: {{ .Release.Name }}
+type: Opaque
+data:
+  DATA_DB_USER: {{ index .Values "cloudsql-proxy" "credentials" "username" | b64enc }}
+  DATA_DB_PASS: {{ index .Values "cloudsql-proxy" "credentials" "password" | b64enc }}
+  DATA_DB_NAME: {{ index .Values "cloudsql-proxy" "credentials" "dbname" | b64enc }}
+  SENTRY_DSN_KEY: {{ .Values.sentry.dsnKey | b64enc }}
+---
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: {{ template "studio.fullname" . }}-clear-old-tasks-cronjob
+  labels:
+    tier: job
+    chart: {{ .Chart.Name }}
+    release: {{ .Release.Name }}
+spec:
+  schedule: "@midnight"
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          restartPolicy: OnFailure
+          containers:
+          - name: app
+            image: {{ .Values.studioApp.imageName }}
+            command:
+            - bash
+            - -c
+            - 'echo "from contentcuration.models import Task; import datetime; Task.objects.filter(created__lt=datetime.datetime.now() - datetime.timedelta(days=7)).delete()" | python contentcuration/manage.py shell'
```

Comment:
@rtibbles any issues with this here python query? I tested this on develop to ensure that we don't accidentally clear any unwanted queries, but want your extra eyes just to be sure 👀

Reply:
That seems correct - for assurance, we could maybe do this as a management command and add a Python unit test for it?